samples/authSample.html

```diff
@@ -37,42 +37,45 @@
 
     <script type="text/javascript">
       // Enter an API key from the Google API Console:
-      //   https://console.developers.google.com/apis/credentials?project=_
+      //   https://console.developers.google.com/apis/credentials
       var apiKey = 'YOUR_API_KEY';
 
+      // Enter the API Discovery Docs that describes the APIs you want to
+      // access. In this example, we are accessing the People API, so we load
+      // Discovery Doc found here: https://developers.google.com/people/api/rest/
+      var dircoveryDocs = ["https://people.googleapis.com/$discovery/rest?version=v1"];
+
       // Enter a client ID for a web application from the Google API Console:
       //   https://console.developers.google.com/apis/credentials?project=_
       // In your API Console project, add a JavaScript origin that corresponds
       //   to the domain where you will be running the script.
       var clientId = 'YOUR_WEB_CLIENT_ID.apps.googleusercontent.com';
 
       // Enter one or more authorization scopes. Refer to the documentation for
-      // the API or https://developers.google.com/identity/protocols/googlescopes
+      // the API or https://developers.google.com/people/v1/how-tos/authorizing
       // for details.
       var scopes = 'profile';
```

Comment:
As profile is explicitly defined in the OpenID spec, I imagine we will continue to support that in perpetuity, but I'll check with the owners of our authorization endpoint.